### PR TITLE
[TR] PIM-4745: Add new style for the notifications

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/status-switcher.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <a class="btn dropdown-toggle btn-status" data-toggle="dropdown" href="#">
     <span class="product-status" data-status="<%= status ? 'enabled' : 'disabled' %>"><%= status ? _.__('pim_enrich.entity.product.btn.enabled') :  _.__('pim_enrich.entity.product.btn.disabled') %></span>
-    <span class="caret"></span>
+    <i class="icon-caret-down"></i>
 </a>
 <ul class="dropdown-menu pull-right">
+    <li></li>
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/status-switcher.html]: you can remove that now :) I fixed it on my PR

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 define(
-    ['backbone', 'jquery', 'underscore', 'routing', 'oro/navigation'],
-    function (Backbone, $, _, Routing, Navigation) {
+    ['backbone', 'jquery', 'underscore', 'routing', 'oro/navigation', 'text!pim/template/notification/notification-list'],
+    function (Backbone, $, _, Routing, Navigation, notificationListTpl) {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: you can name it template as there is only one template in this file

**Comment by sumbobyboys** [comment]
GTM after comment fixes

**Comment by nidup** [comment]
GTM

**Pull request merged**